README.md

```diff
@@ -70,3 +70,21 @@ Can be found in framework-cloud-api, class org.kie.cloud.api.deployment.constant
 | org.kie.server.pwd     | usetheforce123@ | Kie server password  |
 | org.kie.workbench.user | adminUser       | Workbench user       |
 | org.kie.workbench.pwd  | adminUser1!     | Workbench password   |
+
+## Manual template installation
+
+Here you can find steps for installing and initializing Kie template to any OpenShift instance.
```

Comment:
Please use consecutive numbers for steps. Although using 1. for each step might pass as valid markdown, when reading this in plaintext form, it's easy to lose track of the steps.